circle.yml

```diff
@@ -1,6 +1,11 @@
 machine:
-  php:
-    version: 5.6.14
+  xcode:
+    version: 8.2
+
+dependencies:
+  pre:
+    - brew tap homebrew/homebrew-php
+    - brew install php56
```

Comment:
yep, as long as we could have only one compiler version let it be newest for biggest code execution